mappings/net/minecraft/UnusedClass.mapping

```diff
@@ -0,0 +1 @@
+CLASS net/minecraft/class_2081 net/minecraft/UnusedClass
```

Comment:
Are these Annotations? we should probably call them something else.

Reply:
These class aren't used anywhere so idk.